trunk/NDHMS/Land_models/NoahMP/run/MPTABLE.TBL

```diff
@@ -471,3 +471,60 @@ GRAINPT_S8 =    0.0,    0.0,    0.0,    0.0,    0.0,
 BIO2LAI   =  0.035,  0.015,  0.015,  0.015,  0.015,  ! leaf are per living leaf biomass [m^2/kg]
 
 /
+
+&noahmp_optional_parameters
```

Comment:
Really sticking with the text tables.

Reply:
NetCDF someday, maybe. Namelists are a nice way to avoid dependencies...

Reply:
Is there a reason to use unit=15? My only concern here is that 15 is used by WCOSS as the standard output unit. If we know for sure that this will never be used there, not a problem, but otherwise could be an issue.

Reply:
@rcabell No reason to use unit=15. Just looked back through old code and it's been that since the first version I have from 2011.